src/content/en/fundamentals/media/manipulating/files.md

```diff
@@ -0,0 +1,331 @@
+resolutions.
+
+| Abbreviation | Dimensions |
+| ------------ | ---------- |
+| 2160p | 3840x2160 |
+| 1440p | 2560x1440 |
+| 1080p | 1920x1080 |
+| 720p | 1280x720 |
+| 480p | 854x480 |
+| 360p | 640x360 |
+| 240p | 426x240 |
+
+Which one do I use? That depends on your application. For simple embedding you
+may chose a single resolution. If you're preparing files for DASH or HLS, you
+may chose one, several, or all. Fortunately, this is one of the simplest
+transformations you'll make with ffmpeg.
+
+    ffmpeg -i myvideo.webm -s 1920x1080 myvideo_1980x1020.webm
+
+## Encryption
+
+If you plan to enforce copyright on your media, you'll want to encrypt them. I'm
+going to introduce two encryption methods. The first is
+[Widevide](https://www.widevine.com/wv_drm.html), a proprietary encryption
+method offered by Google. The second, also free, is the
+[Clear Key encryption system](https://www.w3.org/TR/encrypted-media/#clear-key)
+, which is supported by a W3C spec.
+
+### Clear Key Encryption
+
+[Clear Key](https://www.w3.org/TR/encrypted-media/#clear-key)
+is a common key system defined in the Encrypted Media Extensions specification
+that uses unencrypted keys to decrypt
+[MPEG Common Encryption (CENC)](https://en.wikipedia.org/wiki/MPEG_Common_Encryption)
+content. 
+
+In practice this is a simple way to encrypt and decrypt content for playback on
+the web. What's more, playback can be done with either DASH or HLS.  This
+section will show you how to encrypt using clear key.
+
+The first thing you need to do is generate a key. You can use whatever method
+you want, but I'm going to use [OpenSSL](https://www.openssl.org/) to write
+sixteen random hex digits to a file. This step is necessary when preparing
+resources for both HLS and DASH.
+
+    openssl rand -out media.key 16
+
+Some versions of OpenSSL seem to create a file with white space and new line
+characters. Make sure these are absent or removed from `media.key` before
+proceeding.
+
+Use Shaka Packager to do the actual encryption. Use the content of the
+`media.key` file for both the `-key` and `-key-id` flags.
+
+Note: Technically, the key ID is supposed to be either the first 8 OR the
+first 16 hex digits of the key. Since packager requires the key to be 16
+digits and does not allow a 32 digit key, both flags use the same value.
+
+    packager \
+      input=myvideo.mp4,stream=audio,output=glocka.m4a \
+      input=myvideo.mp4,stream=video,output=glockv.mp4 \
+      --enable_fixed_key_encryption --enable_fixed_key_decryption \
+      -key INSERT_KEY_HERE -key_id INSERT_KEY_HERE
+
+This example implies that I've typed or pasted the key into the command line
+manually. There's no reason you can't get fancy.
+ 
+    packager \
+      input=myvideo.mp4,stream=audio,output=glocka.m4a \
+      input=myvideo.mp4,stream=video,output=glockv.mp4 \
+      --enable_fixed_key_encryption --enable_fixed_key_decryption \
+      -key "$(< media.key)" -key_id "$(< media.key)"
+
```

Comment:
FYI, since this is useful for every step in the process, I've added a short section called "Check your work" at the beginning of the article.